fe/fe-core/src/main/java/com/starrocks/common/util/RuntimeProfile.java

```diff
@@ -428,7 +427,7 @@ public String getInfoString(String key) {
         return infoStrings.get(key);
     }
 
-    // Merge all the isomorphic sub profiles and the caller must known for sure
+    // Merge all the isomorphic sub profiles and the caller must know for sure
     // that all the children are isomorphic, otherwise, the behavior is undefined
     // The merged result will be stored in the first profile
     public static void mergeIsomorphicProfiles(List<RuntimeProfile> profiles) {
```

Comment:
The FE `mergeIsomorphicProfiles` implementation is same with `merge_isomorphic_profiles`?

Reply:
Yes, exactly the same.